cpp/ql/test/experimental/query-tests/Security/CWE/CWE-295/CurlSSL.cpp

```diff
@@ -0,0 +1,31 @@
+#include<iostream>
+#include<memory>
+
+#include<curl/curl.h>
```

Comment:
We don't use library headers in our language tests (to ensure that tests work and work the same on different machines).  Instead we need so stub the functions and types we're using below so that the code compiles with no headers included.  Something like:
```
struct Curl {
};
typedef Curl CURL;

CURL *curl_easy_init();
void curl_easy_cleanup(CURL *handle);
...
```

You can find stubs for `std::string` and `std::unique_ptr` that should hopefully be sufficient in `cpp/ql/test/library-tests/string_concat/stl.h`.